source/auth/tests/mongodb-oidc.md

```diff
@@ -152,71 +141,58 @@ ______________________________________________________________________
 Drivers that support the [Human Authentication Flow](../auth.md#human-authentication-flow) MUST implement all prose
 tests in this section. Unless otherwise noted, all `MongoClient` instances MUST be configured with `retryReads=false`.
 
-The human workflow tests MUST only be run when testing in the default environment described beflow.
+The human workflow tests MUST only be run when in `ENVIRONMENT:test`.
 
 > [!NOTE]
 > For test cases that create fail points, drivers MUST either use a unique `appName` or explicitly remove the fail point
 > after the test to prevent interaction between test cases.
 
 Drivers MUST be able to authenticate against a server configured with either one or two configured identity providers.
 
-Note that typically the preconfigured Atlas Dev clusters are used for testing, in Evergreen and locally. The URIs can be
-fetched from the `drivers/oidc` Secrets vault, see
-[vault instructions](https://wiki.corp.mongodb.com/display/DRIVERS/Using+AWS+Secrets+Manager+to+Store+Testing+Secrets).
-Use `OIDC_ATLAS_URI_SINGLE` for `MONGODB_URI_SINGLE` and `OIDC_ATLAS_URI_MULTI` for `MONGODB_URI_MULTI`. Currently the
-`OIDC_ATLAS_URI_MULTI` cluster does not work correctly with fail points, so all prose tests that use fail points SHOULD
-use `OIDC_ATLAS_URI_SINGLE`.
-
-If using local servers is preferred, using the
-[Local Testing](https://github.com/mongodb-labs/drivers-evergreen-tools/blob/master/.evergreen/auth_oidc/README.md#local-testing)
-method, use `mongodb://localhost/?authMechanism=MONGODB-OIDC` for `MONGODB_URI_SINGLE` and
-`mongodb://localhost:27018/?authMechanism=MONGODB-OIDC&directConnection=true&readPreference=secondaryPreferred` for
-`MONGODB_URI_MULTI` because the other server is a secondary on a replica set, on port `27018`.
+Unless otherwise specified, use `MONGODB_URI_SINGLE` and the `test_user1` token in the `OIDC_TOKEN_DIR` as the
+"access_token", and a dummy "refresh_token" for all tests.
 
-The default OIDC client used in the tests is configured with `MONGODB_URI_SINGLE` and a valid human callback handler
-that returns the `test_user1` local token in `OIDC_TOKEN_DIR` as the "access_token", and a dummy "refresh_token".
+When using an explicit username for the client, we use the token name and the domain name given by `OIDC_DOMAIN`, e.g.
+`test_user1@${OIDC_DOMAIN}`.
 
 ### (1) OIDC Human Callback Authentication
 
 Drivers MUST be able to authenticate using OIDC callback(s) when there is one principal configured.
 
 **1.1 Single Principal Implicit Username**
 
-- Create default OIDC client with `authMechanism=MONGODB-OIDC`.
+- Create default client
```

Comment:
Is it correct that a "default client" is the same as an "OIDC configured client"? If so, suggest using the more specific "OIDC configured client" term throughout, including the "create a client" statements below.

Reply:
Updated for consistency.